packages/react/src/MinimalFooter/MinimalFooter.features.stories.tsx

```diff
@@ -61,9 +61,10 @@ export const MaximumLinks = () => (
 )
 MaximumLinks.storyName = 'Maximum Links (5)'
 
-export const ButtonLinks = () => (
+export const MixedButtonsAndLinks = () => (
   <MinimalFooter socialLinks={false}>
-    <MinimalFooter.Link as="button" onClick={() => alert('You have clicked Button one')}>
+    <MinimalFooter.Link>Link one</MinimalFooter.Link>
+    <MinimalFooter.Link as="button" href="true" onClick={() => alert('You have clicked Button one')}>
```

Comment:
What is the a href="true" for? It looks like the link above it is also missing a href="#" to make this a mixed use story?

Reply:
This has been updated in this [SHA](https://github.com/primer/brand/pull/280/commits/9ce80ac7b2f3c052c64d3d432177522034703537)

Reply:
Left a [follow up comment](https://github.com/primer/brand/pull/280#discussion_r1202097001) here.